server/handlers/stakeholderHandler.js

```diff
@@ -33,214 +16,230 @@ const stakeholderGetQuerySchema = Joi.object({
   email: Joi.string(),
   phone: Joi.string(),
   website: Joi.string(),
-  children: Joi.array(),
-  parents: Joi.array(),
-  users: Joi.array(),
-  filter: Joi.object(),
+  search: Joi.string(),
+}).unknown(false);
+
+const stakeholderPostSchema = Joi.object({
+  type: Joi.string(),
+  email: Joi.string(),
+  phone: Joi.string(),
+}).unknown(false);
+
+const stakeholderDeleteSchema = Joi.object({
+  id: Joi.string().uuid(),
+  type: Joi.string(),
+  linked: Joi.boolean(),
+  data: Joi.object({
+    id: Joi.string().uuid().required(),
+    email: Joi.string().email(),
+    first_name: Joi.string(),
+    last_name: Joi.string(),
+    org_name: Joi.string(),
+    logo_url: Joi.string(),
+    map: Joi.string(),
+    phone: Joi.string(),
+    website: Joi.string().uri(),
+    children: Joi.array().items(Joi.object()),
+    parents: Joi.array().items(Joi.object()),
+    image_url: Joi.string().allow(''),
+    type: Joi.string(),
+    created_at: Joi.string(),
+    updated_at: Joi.string(),
+  })
+    .unknown(false)
+    .xor('org_name', 'first_name')
+    .xor('org_name', 'last_name'),
 }).unknown(false);
 
-const stakeholderGetAll = async (req, res, next) => {
-  const filter = req.query.filter ? JSON.parse(req.query.filter) : {};
-  const query = { ...req.query, filter };
-  await stakeholderGetQuerySchema.validateAsync(query, {
+const updateStakeholderSchema = Joi.object({
+  id: Joi.string().uuid().required(),
+  email: Joi.string().email(),
+  org_name: Joi.string(),
+  first_name: Joi.string(),
+  last_name: Joi.string(),
+  logo_url: Joi.string(),
+  map: Joi.string(),
+  phone: Joi.string(),
+  website: Joi.string().uri(),
+  children: Joi.array().items(Joi.object()),
+  parents: Joi.array().items(Joi.object()),
+  image_url: Joi.string().allow(''),
+  type: Joi.string(),
+})
```

Comment:
I think that with .unknown(false) we need to add `created_at: Joi.string(), updated_at: Joi.string(),`  here as well